server/src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -527,8 +538,6 @@ public ClusterInfo getClusterInfo() {
                 estimatedHeapUsages.put(nodeId, new EstimatedHeapUsage(nodeId, maxHeapSize.getBytes(), estimatedHeapUsage));
             }
         });
-        final Map<String, NodeUsageStatsForThreadPools> nodeThreadPoolUsageStats = new HashMap<>();
-        nodeThreadPoolUsageStatsPerNode.forEach((nodeId, nodeWriteLoad) -> { nodeThreadPoolUsageStats.put(nodeId, nodeWriteLoad); });
```

Comment:
This appeared to be just a copy, which already happens in the `ClusterInfo` constructor